lib/file-stream.js

```diff
@@ -12,9 +11,9 @@ const debug = debugFactory('webtorrent:file-stream')
  * @param {number} opts.start stream slice of file, starting from this byte (inclusive)
  * @param {number} opts.end stream slice of file, ending with this byte (inclusive)
  */
-class FileStream extends stream.Readable {
+class FileStream extends Readable {
   constructor (file, opts) {
-    super(opts)
+    super(opts ?? {})
```

Comment:
```suggestion
    super(opts)
```

Reply:
issue with this, is that it allows null, and http server creates null ranges, which will error this

Reply:
compare 1st vs 2nd commit, look at the test fail